/**
 * Copyright (c) 2015 - The CM Authors <legal@clickmatcher.com>
 *   All Rights Reserved.
 *
 * This file is CONFIDENTIAL -- Distribution or duplication of this material or
 * the information contained herein is strictly forbidden unless prior written
 * permission is obtained.
 */
package cm;

message FeedConfig {
  required string customer = 1;
  required string feed = 2;
  required uint64 partition_size = 3;
  required uint64 first_partition = 4;
  required uint64 num_shards = 5;
}

